cfme/tests/cloud/test_providers.py

```diff
@@ -303,6 +305,36 @@ def test_openstack_provider_has_api_version():
     assert view.api_version.is_displayed, "API version select is not visible"
 
 
+@pytest.mark.tier(3)
+def test_select_key_pair_none_while_provisioning(request, has_no_cloud_providers):
+    """
+        GH Issue: https://github.com/ManageIQ/manageiq/issues/10575
+
+        Requirement: Have an ec2 provider with single key pair
+                    (For now available in South America (Sao Paulo) region)
+        1. Compute -> Cloud -> Instances
+        2. Click on Provision Instances in Toolbar
+        3. Go to Properties
+        4. Select None in Guest Access Key Pair
+        5. None should be selected
+    """
+
+    prov_config = conf.cfme_data['management_systems']['ec2_sa-east-1']
```

Comment:
I don't like seeing hardcoded keys like `ec2_sa-east-1` here, can this be solved by our provider parametrizer?

Reply:
hm, not sure here, as this region are not commonly used in our tests

Reply:
Updated provider use, will update only region that needed for a test